packages/plugin/local-electron/test/LocalElectronPlugin_spec.ts

```diff
@@ -53,7 +43,11 @@ describe('LocalElectronPlugin', () => {
     beforeEach(() => {
       p = new LocalElectronPlugin({ electronPath: 'test/foo' });
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      p.init('', {} as any);
+      p.init();
+    });
+
+    after(() => {
+      delete process.env.ELECTRON_OVERRIDE_DIST_PATH;
```

Comment:
This is duplicated with the existing code. See: https://github.com/electron/forge/blob/a62c2a15365ca83bb259788eeb71f77bf23ed3b7/packages/plugin/local-electron/test/LocalElectronPlugin_spec.ts#L16-L18

Reply:
It is not, because the delete you referenced is in the other scope.

Reply:
Ah, my bad for not catching that! Would it be better to use `afterEach`?